swagger.json

```diff
@@ -4254,15 +4254,29 @@
                         "PENDING",
                         "ACCEPTED",
                         "REJECTED",
+                        "QUEUED",
```

Comment:
Do we need to add "ANNOTATED"?

Reply:
ah Faiz has an open PR for this: https://github.com/samsarahq/api-docs/pull/140

these changes shouldn't be merged until 12/10, so we should probably keep our changes apart in case we want to update docs before then